GOVERNANCE.md

```diff
@@ -9,9 +9,9 @@ An up to date list of the maintainers can be found at the [Eclipse project page]
 
 ## Monthly meetup
 
-The developer meetup is held monthly on the first Thursday from 17:00 - 18:00 CET. Everyone is welcome to join.
+The developer meetup is held monthly on the first Tuesday from 17:00 - 18:00 CET. Everyone is welcome to join.
```

Comment:
@lucabart97 @gpalmer-latai @Dennis40816 since you are new contributors, this might also be interesting for you to get in touch with the core team :)